src/wallet.js

```diff
@@ -51,40 +56,28 @@ function WalletContextProvider({ children }) {
     }
 
     const walletWeb3 = getWeb3(ethereum)
-    setWalletWeb3(walletWeb3)
-
-    walletWeb3.eth.net
-      .getNetworkType()
-      .then(networkType => {
-        if (!cancel) {
-          setNetworkType(networkType)
-          LocalStorageWrapper.setPrefix(networkType)
-        }
-        return null
-      })
-      .catch(() => {
-        setNetworkType(NETWORK_TYPE_DEFAULT)
-        LocalStorageWrapper.setPrefix(NETWORK_TYPE_DEFAULT)
-      })
+    if (!cancel) {
+      setWalletWeb3(walletWeb3)
+      LocalStorageWrapper.setPrefix(networkType)
+    }
 
     return () => {
       cancel = true
       setWalletWeb3(null)
-      setNetworkType(NETWORK_TYPE_DEFAULT)
       LocalStorageWrapper.setPrefix(NETWORK_TYPE_DEFAULT)
     }
-  }, [ethereum, chainId])
+  }, [ethereum, networkType])
 
   const wallet = useMemo(
     () => ({
       account,
       balance: new BN(filterBalanceValue(balance)),
       ethereum,
-      networkType: connected ? networkType : NETWORK_TYPE_DEFAULT,
+      networkType,
```

Comment:
If the user is not connected, networkType will be undefined. Why don't we set it to `main` as default ? otherwise, how does it work at all in the first place?   because if networkType is undefined, how do you show daos from mainnet when user is not connected ?

If you agree with me,  it can be changed to 

```
networkType: connected ? networkType : KNOWN_CHAINS.get(1),
```

Reply:
yeah, networkType is defined to map to networkName if connected. I can update it to use KNOWN_CHAINS...